### PR TITLE
Client portfolio cases

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
This PR's aims to introduce a portfolio with cases. 

## Preview

screen shot 2018-01-15 at 16 47 33

## Todo

- [ ] ~Conversion form~
- [x] Mobile styling (a bit)
- [x] Case sidebar
- [ ] ~Case next/prev or related or something~
- [x] Real content
- [x] Cases overview page

### CODE REVIEW

File: source/stylesheets/_cases.scss [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    &:hover {
+      .case-inner {
+        background-color: rgba(51, 51, 51, 0.5);
+      }
+    }
+  }
+
+  .case-inner {
+    align-items: center;
+    background-color: rgba(51,51,51,.65);
```
[the remark this answers is not in the document]
Comment (RonaldDijkstra) [reply source/stylesheets/_cases.scss]: image

[the remark this answers is not in the document]
Comment (RonaldDijkstra) [reply source/stylesheets/_cases.scss]: ;-)

**Comment by sn3p** [comment]
Almost ready to merge this awesomeness? Let me know when read for review.

**Comment by RonaldDijkstra** [comment]
There's still plenty to do: 

- [ ] Conversion form
- [ ] Mobile styling (a bit)
- [ ] Real content
- [ ] Cases overview page

**Comment by sn3p** [comment]
> - [ ] Conversion form
> - [ ] Case sidebar
> - [ ] Case next/prev or related or something
> - [ ] Cases overview page

Why not do all these in separate PRs?

**Comment by RonaldDijkstra** [comment]
Conversion form, next prev can be in another PR, but overview is needed at the start.

**Comment by RonaldDijkstra** [comment]
Tested in latest greatest Chrome, FF & Safari + IE10, IE11 & Edge + Mobile safari & chrome

**Comment by sn3p** [comment]
Are these both used?

- source/images/cases/zinn/mood.jpg
- source/images/moods/zinn.jpg

File: locales/nl.yml [hunk 2]
```diff
@@ -116,6 +116,19 @@ nl:
...
+      products: "Producten & Diensten"
+      facts:
+        title: "Facts & Figures"
+        branch: "Branche"
+        since: "Klant sinds"
+        employees: "Aantal medewerkers"
+        website: "Website"
+    overview:
+      title: "Met trots gepresenteerd"
+      text: "Elke klant is anders en heeft een ander eisenpakket en daarom is geen implementatie hetzelfde. Hier een greep uit onze klanten, waarmee we samen goede resultaten hebben behaald."
```
Comment (sn3p) [inline locales/nl.yml]: Suggestie:

```
Elke klant is anders en heeft andere wensen, daarom is elke implementatie weer een uitdaging.
```

File: locales/nl.yml [hunk 3]
```diff
@@ -116,6 +116,19 @@ nl:
...
+      contact_us: "Geïnspireerd door het lezen van onze cases? Of heb je nog meer vragen? Dan is nu het moment om contact met ons op te nemen."
```
Comment (sn3p) [inline locales/nl.yml]: Laatste zin beetje meh?

```
Neem dan contact met ons op.
```

**Comment by RonaldDijkstra** [comment]
label f(#*@$(8 up

**Comment by sn3p** [comment]
Minor and non blocking, but `/klanten` and `/cases` have different spacing between title and paragraph.

**Comment by sn3p** [comment]
screen shot 2018-01-25 at 16 26 00

I think `Hoofd Opleiding & Ontwikkeling` could be a bit smaller and more meta (grey?).

**Comment by RonaldDijkstra** [comment]
#1053

**Comment by RonaldDijkstra** [comment]
Gray is good, put it in the issue.

**Comment by RonaldDijkstra** [comment]
Thx

**Pull request merged**